src/algorithms.jl

```diff
@@ -869,3 +869,16 @@ struct BAOAB{T} <: StochasticDiffEqAlgorithm
   scale_noise::Bool
 end
 BAOAB(;gamma=1.0, scale_noise=true) = BAOAB(gamma, scale_noise)
+
+struct ABOBA{T} <: StochasticDiffEqAlgorithm
+  gamma::T
+  scale_noise::Bool
+end
+ABOBA(;gamma=1.0, scale_noise=true) = ABOBA(gamma, scale_noise)
+
+
+struct OBABO{T} <: StochasticDiffEqAlgorithm
```

Comment:
Can you add the same docstring as for BAOAB for ABOBA and OBABO?

Reply:
I will